### ISSUE TITLE
wrapPython mangles some shebang lines

### ISSUE THREAD

**Issue by edolstra** [NixOS/nixpkgs#9593]
For example, it will rewrite
```
#! /usr/bin/env python2.7
```
to
```
#! /nix/store/ys8vhhxncry5plwzma27vdgz5cv59f2s-python-2.7.10/bin/python2.72.7
```
(note `2.72.7` at the end)

**Comment by domenkozar** [comment]
Sigh.

**Comment by FRidh** [comment]
This was (hopefully) fixed in #14900.
